### PR TITLE
Amobin about

### PR DESCRIPTION
**Pull request by amobin19**
Updating about page with Ayman's and Justin's profile.

### CODE REVIEW

**Comment by Apoorva-A** [comment]
LGTM! Too bad @cyt3ea and I still don't have write access to approve it cough @aw3as cough

**Comment by kienan** [comment]
```
                                                          
88           ,ad8888ba,  888888888888  88b           d88  
88          d8"'    `"8b      88       888b         d888  
88         d8'                88       88`8b       d8'88  
88         88                 88       88 `8b     d8' 88  
88         88      88888      88       88  `8b   d8'  88  
88         Y8,        88      88       88   `8b d8'   88  
88          Y8a.    .a88      88       88    `888'    88  
88888888888  `"Y88888P"       88       88     `8'     88 
```
@Apoorva-A -- awk, @cyt3ea definitely does.

**Comment by Apoorva-A** [comment]
@kienan how you gonna do me like this lol

**Comment by aw3as** [comment]
updated permissions for apoorva + christine

File: app/views/home/about.html.slim [hunk 1]
```diff
@@ -101,6 +101,20 @@
...
         h4.class Class of 2018
+    .col-xs-3.text-center.about-block
+      img =image_tag("about/amobin.jpg")
+      .ustext
+        h4.us Ayman Mobin
+        h5 Web Developer
+      .ushover
+        h4.class Class of 2019
+    .col-xs-3.text-center.about-block
+      img =image_tag("about/jbarry.JPG")
```
Comment (kienan) [inline app/views/home/about.html.slim]: is there a reason JPG is capitalized here?

Comment (amobin19) [inline app/views/home/about.html.slim]: His pic didn't upload until I capitalized them.  Atallah helped us out as we were doing it.

**Comment by cyt3ea** [comment]
In the last push, you also pushed files from the lou folder onto the repo when it should not be there. Also need to update this branch to be up to date with master.

**Comment by kienan** [comment]
Talked via slack -- he's in the process of fixing it and pushing up a corrected version.

Let's make use of the assignee functionality so we can follow the progress. @amobin19 assign me and christine once you're ready for us to look again.

**Pull request merged**